### PR TITLE
feat: add support for simplePDF in Web-Embeds

### PR DESCRIPTION
**Pull request by bendersej**
# Background

This PR adds support for [SimplePDF Embed](https://simplePDF.eu/embed) for the Web-Embeds, [introduced 2 days ago in this PR](https://github.com/excalidraw/excalidraw/commit/b57b3b573da874f7527589c10632ec936c1962d2)

Effectively, this means that one can view, edit, merge, sign PDFs without leaving Excalidraw, by specifying the embed URL: https://embed.simplePDF.eu

A video of SimplePDF in action when embed inside Excalidraw 👇 

https://github.com/excalidraw/excalidraw/assets/10613140/d4e8fd3b-64e3-42be-8e7e-41e31e21d338

# Changes
- Add a new entry to the list of `ALLOWED_DOMAINS`
- Introduce a logic to allow wildcarding the first subdomain: SimplePDF's multi-tenancy is based on the subdomain: `embed` for the free users, and "anything" for users with an account – [read more over here](https://github.com/SimplePDF/simplepdf-embed/blob/main/documentation/IFRAME.md)

# Notes

I took the liberty to add the `allow-downloads` for the iframe attributes, as this is necessary to allow downloading the edited file. 

Let me know if you would want the permission to only be set for `simplePDF`!

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -940,7 +940,7 @@ class App extends React.Component<AppProps, AppState> {
                       title="Excalidraw Embedded Content"
                       allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
                       allowFullScreen={true}
-                      sandbox="allow-same-origin allow-scripts allow-forms allow-popups allow-popups-to-escape-sandbox allow-presentation"
+                      sandbox="allow-same-origin allow-scripts allow-forms allow-popups allow-popups-to-escape-sandbox allow-presentation allow-downloads"
```
Comment (dwelle) [inline src/components/App.tsx:943]: note that the `allow-downloads` isn't supported in Safari

**Review by dwelle — APPROVED** [review]

File: src/components/App.tsx (hunk 1 above)
Comment (bendersej) [reply src/components/App.tsx:944]: Indeed! Luckily not enforced by Safari: it works whether the attribute is present or not

**Comment by bendersej** [comment]
@dwelle, I merge the conflicts!

Thanks for the prompt review!

**Comment by dwelle** [comment]
Thanks! Merging now, we'll release a bit later.

**Pull request merged**